core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1204,6 +1204,10 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-gcp-qe
     namespace: ci
+  - cluster_groups:
```

Comment:
Maybe this should not go together with the QE secrets, as QE uses other accounts to host the ephemeral clusters.

Reply:
Ack. will move it from here.